recipes/finetuning/README.md

```diff
@@ -1,8 +1,8 @@
 # Finetuning Llama
 
-This folder contains instructions to fine-tune Llama 2 on a 
+This folder contains instructions to fine-tune Llama 2 on a
```

Comment:
As we touch the README could you add here that we also support Meta Llama 3 now?

Reply:
I think there are more steps needed to support Meta Llama3 like "[Convert the model weights using Hugging Face transformer from source](https://github.com/meta-llama/llama-recipes/blob/main/recipes/quickstart/Running_Llama3_Anywhere/Running_Llama_on_HF_transformers.ipynb)". @HamidShojanazeri @mreso I think this PR is just for adding new profiling feature, and it is better to make another PR for the Meta Llama3 support?

Reply:
@wukaixingxp we already made the changes for Llama 3 and document it in the main [README](https://github.com/meta-llama/llama-recipes/blob/54d285956653b4ff495bd21dc85957cf58363dd1/README.md?plain=1#L3). But I can create a follow up PR to update this README to keep things separated.

Reply:
I think another PR is needed, as even the current main readme is still about how to [convert llama2-7B](https://github.com/meta-llama/llama-recipes/tree/main?tab=readme-ov-file#model-conversion-to-hugging-face). I can only find conversion instruction for Llama3 [here](https://github.com/meta-llama/llama-recipes/blob/main/recipes/quickstart/Running_Llama3_Anywhere/Running_Llama_on_HF_transformers.ipynb).